### ISSUE TITLE
No dock peeking for minimized or hidden windows - is this by design?

### ISSUE THREAD

**Issue by 0n0mat0p03ia** [ejbills/DockDoor#2]
Dock peeking is unavailable for minimized and hidden windows. Is this by design? Dock peeking would be much more useful if one could peek at hidden and/or minimized windows.

**Comment by ejbills** [comment]
Will look into this.

**Comment by ejbills** [comment]
Fully implemented, will be in 1.0.4.

**Comment by 0n0mat0p03ia** [comment]
> Fully implemented, will be in 1.0.4.

Tested with 1.0.5. 

**Dock Peek** 

- I see the update you made for minimized windows. I'm guessing previews can't be generated for minimized windows

- I don't see any peek previews for windows if an app is hidden. I do see the minimized windows listed in dock peek, however. 
  - For example, if I have 3 windows, I select to minimize 2 windows. When viewing the 3rd window I select to hide the app. In dock peek, I see the 2 minimized windows listed, but nothing for the 3rd window.

**Ctrl+Tab**

- Don't see previews for hidden app windows
- Don't see anything for minimized windows
  - When I first started testing, I thought I saw entries for minimized windows (same as dock peek), but I don't see it now. Maybe I'm misremembering...
